ui-tests/cypress/support/pageObjects/app/AppTour.js

```diff
@@ -17,28 +17,28 @@ export default class AppTour extends WebElement {
                .find(this.tourContainer);
   }
 
-  #getFooter() {
+  getFooter() {
```

Comment:
Why did you removed the private modifier? Not really needed, but did you needed this method outside?

Reply:
it was preventing the cypress to run here, I was getting a message saying that the private modifier # is not supported. I'll try to add it back and see if it works